modules/webstyle/lib/webstyle_templates.py

```diff
@@ -939,7 +945,7 @@ def detailed_record_container_bottom(self, recid, tabs, ln=CFG_SITE_LANG,
     <br/>
     """ % {'similar' : similar,
            'dates' : creationdate and '<div class="recordlastmodifiedbox" style="position:relative;margin-left:1px">&nbsp;%(dates)s</div>' % {
-                'dates': _("Record created %(x_date_creation)s, last modified %(x_date_modification)s") % \
+                'dates': _("Record added %(x_date_creation)s, last modified %(x_date_modification)s") % \
                 {'x_date_creation': creationdate,
```

Comment:
> amend the branch

(To clarify, I rather meant to create a new branch/PR, keeping only 005 parts out of this branch/PR; then we could close this PR.)